types.py

```diff
@@ -0,0 +1,11 @@
+# ideia de centralizar todas os tipos aqui. 
+# Para no retorno das funções ao invés de por exemplo "INT"
+# colocar "TYPE.INT"
+
+class TYPE():
+    INT = "INT"
+    BOOL = "BOOL"
+    ERROR = "ERROR"
+    
+    def FUNC(type_param, type_body):
+        return "(" + type_param + ")" + " -> " + type_body
```

Comment:
I have some doubts but can we be sure EVERY inference answer is going to be this way?
Considering all possibilities and cases?

Reply:
Vamo falar em port ahhahaha! Tipo, não se trata de uma inferência, elas nem fazem parte do escopo do trabalho. Esse enum serve para representar os tipos possíveis de qualquer expressão da L1, que podem ser int, bool ou "tipo função". O " tipo função" é escrito desta forma:  <tip-do-parametro> -> <tipo-do-corpo>

Reply:
A ideia dessa função é justamente receber esses dois tipos (o do parametro e o do corpo) criar uma string pra representar um tipo função

Reply:
Só pra ter ctz que tamos alinhados:

Um exemplo de termo do tipo função: "fn y: int => false". Essa expressão é do tipo "int -> bool", visto que o param (y) é um int e o retorno (false) é um bool